### DISCUSSION TITLE
Help with contributing to Zed

### DISCUSSION THREAD

**Discussion by dhruvkelawala** [zed-industries/zed#9864] [Feature Requests]
I really like using Zed and would like to contribute to it. I have worked with Rust but not with a code editor before. I have given the contribution doc a read but it doesn't specify the issues I can start working with. I think it will be a good idea to mark some issues as "good first issue". However, in absence of that, can someone from the dev team guide me on where I could start contribution from? Interested in taking up AI related tasks and auto-closing HTML/JSX tags. 

Thanks

**Comment by Moshyfawn** [comment]
There's actually a [`good first issue`](https://github.com/zed-industries/zed/labels/good%20first%20issue) tag you can check out 🥳

As for the roadmap and the current priorities you can jump on to help with, [the "Contribution Guide" section](https://github.com/zed-industries/zed/blob/main/CONTRIBUTING.md#contribution-ideas) has links to both [the roadmap](https://zed.dev/roadmap) and [the current top issues](https://github.com/zed-industries/zed/issues/5393) we'd like your help on.

**Comment by JosephTLyons** [comment]
Also, another way to contribute is by building and maintaining packages. In the README of the [extensions repo](https://github.com/zed-industries/extensions), there are links to bring up the most popular language and theme requests in the GitHub Issues tracker.
